### PR TITLE
[js/webgpu] generate operator table for webgpu

### PR DESCRIPTION
**Pull request by fs-eire**
### Description
[js/webgpu] generate operator table for webgpu

### CODE REVIEW

File: js/web/docs/webgpu-operators.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## Operators Support Table
+
+The following table shows ONNX
+operators from which onnx opset version are currently supported in WebGPU EP by ONNX Runtime Web. For example,
+`4-6, 8+` means ONNX Runtime Web currently support opset version 4 to 6, 8 and above.
+
+*This file is automatically generated from thedef files via [this script](../script/generate-webgpu-operator-md.ts).Do not modify directly.*
```
Comment (satyajandhyala) [inline js/web/docs/webgpu-operators.md:7]: Nit. "the def" instead of "thedef"

File: js/web/docs/webgpu-operators.md
Comment (satyajandhyala) [inline js/web/docs/webgpu-operators.md:1]: Do we need copy rights at the top of the file even if it is automatically generated?

File: js/web/docs/webgpu-operators.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+## Operators Support Table
+
+The following table shows ONNX
+operators from which onnx opset version are currently supported in WebGPU EP by ONNX Runtime Web. For example,
```
Comment (satyajandhyala) [inline js/web/docs/webgpu-operators.md:4]: How about?
"The following table shows ONNX operators and the supported opset-versions in WebGPU EP by ONNX Runtime Web"

File: js/web/docs/webgpu-operators.md
Comment (fs-eire) [reply js/web/docs/webgpu-operators.md:1]: I don't think so. Regardless of whether it's generated or not, we generally don't put copyright banners on markdown files in this repo.

**Review by guschmue — APPROVED** [review]

**Pull request merged**